CMakeLists.txt

```diff
@@ -2,7 +2,7 @@ cmake_minimum_required(VERSION 3.12.0)
 
 project(ni_grpc_device_server
   LANGUAGES C CXX
-  VERSION 1.5.2)
+  VERSION 2.0.0)
```

Comment:
@harsha-bhushan , I think we had a discussion at some point on the next version of the grpc-device release but I wanted to point it out here. I believe the most substantial reason to bump the major version this release is because of the overhaul in how error-reporting is done with Dan's PR: https://github.com/ni/grpc-device/pull/692.

Do you agree that the next release should be 2.0.0 instead of a minor version bump to 1.6.0?

Reply:
I agree